sklearn/model_selection/_split.py

```diff
@@ -1578,6 +1605,12 @@ def split(self, X, y, groups=None):
 
         test : ndarray
             The testing set indices for that split.
+
+        Note
+        ----
+        Randomized CV splitters may return different results for each call of
+        split. This can be avoided (and identical results returned for each
```

Comment:
This seems a bit redundant. Maybe "You can make the results identical by setting random_state to an integer"?